db/migrate/20240207130255_create_roles.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRoles < ActiveRecord::Migration[7.1]
+  def change
+    create_table :roles do |t|
+      t.string :role_name
```

Comment:
```suggestion
      t.string :name
```
Inside Role table, name attribute will itself be self-explanatory